package.json

```diff
@@ -3,32 +3,25 @@
   "version": "4.0.0",
   "description": "DOM plugin for the unexpected assertion libary",
   "main": "lib/index.js",
-  "files": [
-    "lib",
-    "unexpected-dom.js",
-    "unexpected-dom.min.js"
-  ],
+  "files": ["lib", "unexpected-dom.js", "unexpected-dom.min.js"],
```

Comment:
Could we add `.prettierignore` and add `package.json` to it? In my experience, `prettier` will battle `npm install --save`, changing it back and forth.